userguide/image_classifier/README.md

```diff
@@ -30,6 +30,43 @@ data.save('cats-dogs.sframe')
 data.explore()
 ```
 
+#### Annotating Data
```

Comment:
Please don't add before the introductory example. Add this in the advance section or after the introductory example. Let's keep the first example really simple.